src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -382,3 +409,51 @@ private extension UITableViewCell {
 		}
 	}
 }
+
+/// Stuff to draw a background to the cell, depending on the position of the cell in the section it gets rounded corners.
+
+private extension UITableViewCell {
+	enum BackgroundLocation: Int {
+		case top = 200_001
+		case bottom = 200_002
+		case inBetween = 200_003
+	}
+	
+	func removeBackground() {
+		viewWithTag(BackgroundLocation.top.rawValue)?.removeFromSuperview()
```

Comment:
Adding and removing views via tagging makes me a little bit uncomfortable. I know the pattern with the tagged views is copied from the seperators. I'm wondering if there is a better object oriented solution (oh my god, I wrote the forbidden phrase 🙈) Just for discussion.

Reply:
I will think about the forbidden phrase ;)

Reply:
Actually, nothing really good comes to my mind. I will create a discussion for our dev camp.